kalite/settings/base.py

```diff
@@ -274,7 +255,14 @@
 STATIC_ROOT = getattr(local_settings, "STATIC_ROOT", STATIC_ROOT)
 MEDIA_URL = getattr(local_settings, "MEDIA_URL", "/media/")
 STATIC_URL = getattr(local_settings, "STATIC_URL", "/static/")
+
+
 DEFAULT_DATABASE_PATH = getattr(local_settings, "DATABASE_PATH", DEFAULT_DATABASE_PATH)
+
+# This database is located in the content root because then it can be copied
+# together with the other media files located there.
+ASSESSMENT_ITEMS_DATABASE_PATH = os.path.join(CONTENT_ROOT, 'assessmentitems.sqlite')
```

Comment:
I don't understand how this is overridable in project settings, if it's defined here and then used immediately below.